### DISCUSSION TITLE
Daily Standup (Wednesday, June 1st)

### DISCUSSION THREAD

**Discussion by BrandonMagana** [CarlosCRG19/ijalti-peers#82] [General]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:
```
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x 
 ```

**Comment by BrandonMagana** [comment]
**What did you achieve in the last 24 hours?**:
 - Not much, I spent some time working with @antonieto trying to fix auth problems

**What are your priorities for the next 24 hours?**:
 -  Resolve this  auth problems
 - Finish endpoint for aspirant filtering

**Blockers**:
 - Problems with authentication middleware

**Shoutouts**:
 - @antonieto for making progress on the backend

**Comment by DavBot02** [comment]
**What did you achieve in the last 24 hours?**:
 - Cleaned code a bit
 - Skill obtention from endpoint

**What are your priorities for the next 24 hours?**:
 - Be able to post job offer

**Blockers**:
 - Not sure how to handle company id

**Shoutouts**:
 - @BrandonMagana @CarlosCRG19 @antonieto for helping me fix my git mess

**Comment by CristianCazares** [comment]
**What did you achieve in the last 24 hours?**:
 - include some of the auth

**What are your priorities for the next 24 hours?**:
 - Continue with search aspirant

**Blockers**:
 - Elective stuff

**Shoutouts**:
 - @everyone for solving the git mess
